packages/arb-avm-cpp/data_storage/src/arbcore.cpp

```diff
@@ -226,9 +226,14 @@ void ArbCore::printDatabaseMetadata() {
         std::cout << std::get<rocksdb::Status>(checkpoint_result).ToString()
                   << "\n";
     } else {
-        printMachineOutputInfo(
-            "last checkpoint",
-            std::get<MachineStateKeys>(checkpoint_result).output);
+        auto output =
+            getMachineOutput(std::get<CheckpointVariant>(checkpoint_result));
+        if (std::holds_alternative<rocksdb::Status>(checkpoint_result)) {
+            std::cout << std::get<rocksdb::Status>(checkpoint_result).ToString()
+                      << "\n";
```

Comment:
We already checked whether `checkpoint_result` was a status above

Reply:
:+1: